Detectors/EMCAL/base/src/Geometry.cxx

```diff
@@ -897,12 +882,11 @@ void Geometry::CreateListOfTrd1Modules()
   }
 }
 
-ShishKebabTrd1Module* Geometry::GetShishKebabModule(Int_t neta) const
+const ShishKebabTrd1Module &Geometry::GetShishKebabModule(Int_t neta) const
```

Comment:
I think that the `&` should be attached to the left string. There are many places like this.